integration-tests/api/medusa-config.js

```diff
@@ -42,16 +42,14 @@ module.exports = {
     },
     workflows: true,
     [Modules.AUTH]: {
-      scope: "internal",
-      resources: "shared",
-      resolve: "@medusajs/auth",
       options: {
         providers: [
           {
-            name: "emailpass",
-            scopes: {
-              admin: {},
-              store: {},
+            resolve: "@medusajs/auth-emailpass",
+            options: {
+              config: {
+                emailpass: {},
+              },
```

Comment:
I just asked out of curiosity because I wanted to hear your thinking. 

I don't have a strong opinion on whether to have a separate package or not, but the general idea with the local providers (in payments, fulfillment, and other domains, too) has been to reduce friction for the common provider use cases and make it easy to get started. 

We can eliminate this potential friction by installing the package and enabling it in the auth module by default, but it is still more error-prone than just baking it in. If you don't have this configured correctly, the admin is effectively useless, as you cannot sign in. 

That said, let's go with this for now. We can revisit if this becomes a huge pain for devs getting started. I think having a consistent provider architecture should makes a lot of sense. And maybe auth specifically, is a slightly different use case than e.g. payments.

Reply:
Should we migrate google to a package too then?

Reply:
@olivermrbl yup as you can see in the PR description I wrote that as a remaining TODO 😄 

I see where you're coming from, and I do agree that this can cause friction. But I think us having a consistent setup and architecture will allow us to improve the UX of all modules, not just auth, and if we are worried the UX is bad or the possibility for misconfiguring is high, we should focus on fixing that instead. At least that's how I see it 😄

Reply:
> @olivermrbl yup as you can see in the PR description I wrote that as a remaining TODO 😄

Missed that, my bad 😅